lib/utils/normalize-rel-path.ts

```diff
@@ -0,0 +1,5 @@
+export function normalizeRelPath(input: string) {
+  // replaces duplicate slashes with single slash: ////test///1 -> /test/1
+  const output = input.replace(/(\/)\/+/g, '$1');
```

Comment:
another way would be
```suggestion
  const output = input.replace(/\/\/+/g, '');
```